package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@politico/vue-accessible-selects",
-  "version": "0.4.3",
+  "version": "0.4.4",
```

Comment:
Hi @ee923925github - these changes are perfect, but can you remove this version bump here? We'll do a publish after your PR merges